src/coreclr/scripts/superpmi.py

```diff
@@ -1090,6 +1090,11 @@ def replay(self):
             if self.coreclr_args.spmi_log_file is not None:
                 flags += [ "-w", self.coreclr_args.spmi_log_file ]
 
+            if self.coreclr_args.arch == "arm":
```

Comment:
These `-target` flags are only needed for altjit, so I think you need to put them under `if self.coreclr_args.altjit`

Reply:
Agree, let me update this